src/m365/planner/commands/plan/plan-details-get.ts

```diff
@@ -117,25 +111,30 @@ class PlannerPlanDetailsGetCommand extends GraphItemsListCommand<any> {
       });    
   }
 
-  private getPlanDetails(args: CommandArgs): Promise<any> {
+  private getPlanDetails(logger: Logger,args: CommandArgs,cb: () => void): void {
     const requestOptions: any = {
       url: `${this.resource}/v1.0/planner/plans/${args.options.planId}/details`,
       headers: {
-        'accept': 'application/json'
+        'accept': 'application/json;odata.metadata=none'
       },
       responseType: 'json'
     };
 
-    return request.get(requestOptions);
+    request
+      .get<{ value: PlanDetails }>(requestOptions)
+      .then((res: any): void => {
+        logger.log(res);
```

Comment:
Let's keep logging the output and closing the command in the `commandAction` method so that it's easier for everyone to follow. Since we're just executing a single call here, there's really no need to encapsulate it inside this method.

Reply:
done